src/Toml/Printer.hs

```diff
@@ -86,8 +87,12 @@ prettyTomlInd :: PrintOptions -- ^ Printing options
               -> TOML         -- ^ Given 'TOML'
               -> Text         -- ^ Pretty result
 prettyTomlInd options i prefix TOML{..} =
-    prettyKeyValue options i tomlPairs <> "\n"
-    <> prettyTables options i prefix tomlTables
+  Text.intercalate "\n" $ filter (not . Text.null)
+    [ prettyKeyValue options i tomlPairs
+    , prettyTables options i prefix tomlTables
+    , prettyTableArrays options i prefix tomlTableArrays
```

Comment:
Just a minor cosmetic suggestion but I guess that aligning `options`, `i` and other variables under each other will makes this section cleaner 👍